### PR TITLE
Add 'Ruyter & Ast' and 'Ludwig von Kapff'

### PR DESCRIPTION
**Pull request by moriarty-jim**
Added two (similar) wine dealers: "Weingesellschaft Ruyter & Ast GmbH" and "Ludwig von Kapff GmbH".

### CODE REVIEW

File: companies/ludwigvonkapff.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+        "de"
+    ],
+    "categories": [
+        "commerce"
+    ],
+    "name": "Ludwig von Kapff GmbH",
+    "address": "Speicher I\nKonsul-Smidt-Str. 8 J\n28217 Bremen\nDeutschland",
+    "phone": "+49 421 3994317",
+    "fax": "+49 421 3994301",
+    "email": "service@ludwig-von-kapff.de",
```
Comment (baltpeter) [inline companies/ludwigvonkapff.json:13]: The privacy policy lists a better mail (datenschutz@egfra.de). We should use that instead. Then, we can also set the `suggested-transport-medium` to `email`.

File: companies/ruyterast.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        "commerce"
+    ],
+    "name": "Weingesellschaft Ruyter & Ast GmbH",
+    "runs": [
+        "Club of Wine"
+    ],
+    "address": "Speicher I\nKonsul-Smidt-Str. 8 J\n28217 Bremen\nDeutschland",
+    "phone": "+49 421 3053251",
+    "fax": "+49 421 3053259",
+    "email": "service@ruyter-und-ast.de",
```
Comment (baltpeter) [inline companies/ruyterast.json:16]: The datenschutz@egfra.de email also applies here as per the privacy policy.

**Review by baltpeter — CHANGES_REQUESTED** [review]
Thanks for the record! I have requested some changes below.

**Comment by moriarty-jim** [comment]
Thanks for the suggestions! I've updated the files.

**Review by baltpeter — APPROVED** [review]
Thank you!

**Pull request merged**